tests/testcases/040_check-network-adv.yml

```diff
@@ -211,6 +213,8 @@
               command: ["/bin/bash", "-c", "sleep 2000000000000"]
               image: dougbtv/centos-network
           EOF
+      delegate_to: "{{ groups['kube-master'][0] }}"
```

Comment:
Regarding delegate_to+run_once vs when, here is a recommendation from ansible
```
When used together with “serial”, tasks marked as “run_once” will be run on one host in each serial batch. If it’s crucial that the task is run only once regardless of “serial” mode, use when: inventory_hostname == ansible_play_hosts_all[0] construct.
```
So I've always used the when syntax to achieve that use case.

Reply:
Ok I've copied other task truth to be told, but why not follow the best practice indeed